app/javascript/react/containers/app.js

```diff
@@ -11,7 +11,9 @@ const App = props => {
     <Router history={browserHistory}>
       <Route path='/' component={NavBar} >
         <IndexRoute component={WizardsContainer} />
+
       </Route>
+      <Route path='/wizards/:name' component={WizardShowContainer}/>
```

Comment:
Do you not want the NavBar on the show page? If you do, you should nest this route on Line 14.

Reply:
Good call. 
I've changed that and pushed it back up, assuming the tests still pass we're good.